src/core/mcap_tests/cpp/test_mcap_recorder.cpp

```
@@ -1,22 +1,21 @@
 // SPDX-FileCopyrightText: Copyright (c) 2025-2026 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-// Unit tests for McapRecorder
+// Unit tests for McapTrackerChannels<RecordT, DataTableT> and McapRecordingConfig.
+
+#define MCAP_IMPLEMENTATION
 
 #include <catch2/catch_test_macros.hpp>
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)